app/lib/constants/site_config.rb

```diff
@@ -280,7 +280,7 @@ module SiteConfig
       },
       sidebar_tags: {
         description: "Determines which tags are shown on the homepage righthand sidebar",
-        placeholder: "List of valid tags: comma separated, letters only e.g. help,discuss,explainlikeimfive,meta"
+        placeholder: "List of valid, comma-separated tags e.g. help,discuss,explainlikeimfive,meta"
```

Comment:
This field technically accepts numbers and will pin tags to the homepage righthand sidebar that contain numbers (or are made up entirely of numbers, like `11`). We don't suggest that Admins use numbers in the Admin Guide, but we also don't suggest that they use "letters only," so this is more consistent with what the docs say.